control/api_views.py

```diff
@@ -159,7 +165,10 @@ def get_queryset(self):
     def __create_or_update(self, request, save_questionnaire_func, is_update):
         if is_update:
             pre_existing_qr = self.get_object()  # throws 404 if no qr
-            verb = 'updated'
+            if request.data.get('is_draft') is False:
+                verb = 'published'
```

Comment:
Peut etre qu'il faut aussi tester si `pre_existing_qr.is_draft is True` ? 
Parce que je crois que quand tu fais un questionnaire update d'un questionnaire deja publié, tu peux envoyer ce flag is_draft aussi, et ca va etre loggé comme une deuxieme publication. (ok actuellement ya pas de UI qui update un questionnaire deja publié, mais bon c'est ptet plus safe quand meme, non?)

Reply:
ok j'ai rajouté le check